apps/src/componentLibrary/StylizedBaseDialog.jsx

```diff
@@ -70,14 +82,23 @@ export default function StylizedBaseDialog(props) {
 
   return (
     <BaseDialog {...passThroughProps()} useUpdatedStyles>
-      <div style={styles.container}>
-        <h1 style={styles.title}>{props.title}</h1>
+      {props.title && (
+        <>
+          <div style={styles.container}>{renderTitle()}</div>
+          {horizontalRule}
+        </>
+      )}
+      <div
+        style={{
+          ...styles.container,
+          ...(props.type === 'simple' ? {} : {padding: GUTTER})
```

Comment:
Should `{padding: GUTTER}` be move to the `styles` object?

Reply:
oh actually looks like you took it out of the styles object in this change. Out of curiosity, why?

Reply:
yeah, i debated what to do here. it felt incorrect to have `styles.body` continue to define the styles when they would only apply to 1/2 cases (or 1/n as we add more "types"), so i opted for this even though i kind of hate it 😂 

an equivalent solution that may be better here would be something like:

```javascript
// in render
<div style={{...styles.container, ...(styles.body[type] || {})}}

const styles = {
  body: {
    default: {padding: GUTTER}
}
```

_or_ have `styles.body` continue to define padding but have `styles.body.simple` override that padding. thoughts on which is better?

Reply:
What about
```
// in render
<div style={{...styles.container, ...(styles.body[type])}}
const styles = {
  body: {
    default: {padding: GUTTER}
    simple: {}
}
```
?
🤷‍♀️  No strong preferences either way though